java/src/jmri/jmrix/tams/TamsTurnout.java

```diff
@@ -116,7 +116,8 @@ synchronized void setCommandedStateFromCS(int state) {
             log.debug("Returning");
             return;
         }
-        log.debug("Setting to state " + state);
+        forwardCommandChangeToLayout(state);
```

Comment:
The method this is included in is setting the known state based in feedback from the command station.

Forwarding the state back to the command station is unnecessary because the command station was the reporter of the state change.  Also, including this line here can, depending on the protocol, cause an infinite loop of feedback then request.  This line needs to be removed.

Reply:
Line removed, thanks. I had seen it in another xTurnout, but understand it is not a pattern